intake/catalog/base.py

```diff
@@ -61,6 +61,9 @@ def __init__(self, *args, name=None, metadata=None, auth=None, ttl=1,
         name : str, optional
             Unique identifier for catalog. This is primarily useful when
             manually constructing a catalog. Defaults to None.
+        description : str, optional
+            Description of the catalog. This is primarily useful when
```

Comment:
"This is primarily..." - I don't think this is required. Only need say that this takes precedence over whatever is stated in the cat file itself.